boards/xtensa/xt-sim/Kconfig.board

```diff
@@ -5,3 +5,4 @@
 
 config BOARD_XT_SIM
 	bool "Xtensa Development ISS"
+	depends on SOC_XTENSA_SAMPLE_CONTROLLER
```

Comment:
no, don't put this here.
We run xt-sim with a variety of SOC targets, see all the defconfigs in this directory. We shouldn't pin this to a specific SOC, it can really run all of them.

Reply:
ok, I will take it our of this PR, but we need to fix this, I noticed this Xtensa board appearing in menuconfig when configuring an X86 board, it need to depend on something that is Xtensa based.

Reply:
ok, maybe it should depend on SIMULATOR_XTENSA